neo.UnitTests/Ledger/UT_MemoryPool.cs

```diff
@@ -1,19 +1,36 @@
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Moq;
+using Neo.Cryptography;
+using Neo.IO;
+using Neo.IO.Caching;
 using Neo.Ledger;
 using Neo.Network.P2P.Payloads;
 using Neo.Persistence;
+using Neo.Plugins;
+using Neo.SmartContract.Native;
 using System;
+using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
 
 namespace Neo.UnitTests.Ledger
 {
+    internal class TestIMemoryPoolTxObserverPlugin : Plugin, IMemoryPoolTxObserverPlugin
```

Comment:
Where is this plugin used?

Reply:
When the instance of `TestIMemoryPoolTxObserverPlugin` is created, Plugin will add this instance to `TxObserverPlugins` list via `Plugin `constructor. 
The TestIMemoryPoolTxObserverPlugin is used to cover the lines such as https://github.com/neo-project/neo/blob/1a392037f23a767a943c7f05adb0ce0d6916297f/neo/Ledger/MemoryPool.cs#L281-L286